cf-check/dump.c

```diff
@@ -154,23 +155,27 @@ static void print_struct_averages(
     else
     {
         // TODO: clean up Averages
+        Seq *slotsMeta = SeqNew(CF_OBSERVABLES, free);
         const Averages *const averages = value.mv_data;
         const time_t last_seen = averages->last_seen;
 
-        JsonElement *all_observables = JsonObjectCreate(observables_max);
```

Comment:
TODO: make the nice observables_strings a backup in case no ts_key file is provided or found. (aka slotsMeta comes back empty).

Reply:
done.